app/assets/javascripts/sites/edit.js

```diff
@@ -8,15 +8,15 @@ function addSiteImageUriFieldSet(addButtonFieldset) {
 	const newInput = document.createElement("input");
 	newInput['type'] = 'text';
 	newInput['name'] = 'site[image_uris][]';
-	newInput['id'] = `site_image_uris[${newIndex}]`;
+	newInput['id'] = 'site_image_uris[' + newIndex + ']';
 	newFieldset.appendChild(newInput);
 	// <input type="button" class="btn btn-danger" value="Remove" onclick="this.parentElement.remove();"/>
 	if (newIndex > 0) {
 		const newButton = document.createElement("input");
 		newButton['type'] = 'button';
 		newButton['className'] = 'btn btn-danger';
 		newButton['value'] = 'Remove';
-		newButton['onclick'] = () => this.parentElement.remove();
+		newButton['onclick'] = function() { this.parentElement.remove() };
```

Comment:
Yeah, unfortunately without babel we'll need to stick with old fashioned functions for IE 11 compatibility.

Reply:
it was sad times when I deployed the earlier version :-(